lib/shoulda/matchers/active_model/numericality_matchers/even_number_matcher.rb

```diff
@@ -20,6 +20,11 @@ def allowed_type
           def diff_to_compare
             2
           end
+
+          def strict
```

Comment:
Sorry, misspoke here, they don't inherit from ValidationMatcher. We're actually trying to avoid inheriting from ValidationMatcher in the future. What you have here is fine.